CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -671,6 +652,20 @@ function Staff:afterLoad(old, new)
     return
   end
 
+  if old < 162 and new >= 162 then
+    self.leave_priority = AnnouncementPriority.High
```

Comment:
This is subtly different from the actual after state, here you are adding the values to a different table (one level up in the chain). It doesn't matter here, but it could cascade. Suggest you use the afterLoad for each type instead.

Reply:
Each type of staff? That was the cascade I was aiming for.